docs/guide/1.x/setup.md

```diff
@@ -0,0 +1,138 @@
+# Setting up
+Spek is implemented as a [JUnit Platform](https://junit.org/junit5) TestEngine. 
+
+Include `org.jetbrains.spek:spek-api` and `org.jetbrains.spek:spek-junit-platform-engine` 
+in the test classpath. The former is needed during compilation, while the latter during runtime only.
+
+## Gradle
+Gradle `4.7` is recommended as it added a built-in support for JUnit Platform.
+
+```groovy
+// setup dependencies
+dependencies {
+    // some version of Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+
+    testImplementation ('org.jetbrains.spek:spek-api:1.x.x')  {
+        exclude group: 'org.jetbrains.kotlin'
+    }
+    testRuntimeOnly ('org.jetbrains.spek:spek-junit-platform-engine:1.x.x') {
+        exclude group: 'org.junit.platform'
+        exclude group: 'org.jetbrains.kotlin'
```

Comment:
Why do we need to exclude these explicitly?

Reply:
It's for overriding the kotlin version used. Excluding `org.jetbrains.kotlin` and specifying dependencies to `kotlin-stdlib` and `kotlin-reflect` will prevent versions conflicts (`kotlin-reflect` is not backwards compatible).

Reply:
KotlinTest does not require this actually.

Reply:
an alternative to `exclude` could be `force` of particular versions of particular artifacts

https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ResolutionStrategy.html

Reply:
Let me check if I can drop the `kotlin-reflect` requirement, and just use plain old java reflection.